app/[lang]/assets/icons/Logo.tsx

```diff
@@ -0,0 +1,17 @@
+const Logo = () => {
```

Comment:
Would it make sense to move the Logos into the components-folder? Like `app/components/icons`?